lib/builder.js

```diff
@@ -264,7 +264,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   };
 
   // jshint evil:true
-  new Function(source.toString()).call(global);
+  eval(source.toString());
```

Comment:
Well telling people 'your config file must not have `return` at top level' would be quite reasonable IMHO. After all it's supposed to be included as script in web pages:

    <script src="config.js"></script>

is the way to use single `config.js` for both development and production workflow.